Dockerfile.assisted-test-infra

```diff
@@ -1,4 +1,15 @@
-FROM quay.io/edge-infrastructure/assisted-service:latest AS service
+# Generate python client
+FROM quay.io/edge-infrastructure/swagger-codegen-cli:2.4.18 AS swagger_py
+
+COPY assisted-service/swagger.yaml /
+COPY assisted-service/tools/generate_python_client.sh .
```

Comment:
Any reason why we should leave this python generating script in assisted-repo?

Reply:
we need this to stay in assisted-service because we have a postsubmit job in assisted-service that generates python client and push it to pypi

Reply:
the issue I see with this is that 5 lines of bash script create a dependency on having this checkout available. Which might be OK during normal usage as it normally it would be there, but it might not always be the case.

I know we agreed on this approach in general, but this is the reason why I don't like having 2 places where we generate the python client. Can this be simplified?